deno/config.ts

```diff
@@ -1,6 +1,9 @@
-const [functionURL, collectorURL, rawExitCodes] = Deno.args
+const [functionURL, collectorURL, bootstrapURL, rawExitCodes] = Deno.args
 const exitCodes = JSON.parse(rawExitCodes)
 
+const { env } = await import(new URL('./globals.ts', bootstrapURL).toString())
```

Comment:
Doesn't this implicitly make the `globals.ts` an external API? Like, if you rename that file as part of a refactoring in the bootstrap repo and merge the PR, this would silently break?

Could we export `env` from `index-combined.ts`, which is a public entry point, and then import it from there?

Reply:
> Could we export env from index-combined.ts, which is a public entry point, and then import it from there?

That sounds like a good idea, yes. What do you think about exporting `Netlify` instead, so that we can do `globalThis.Netlify = Netlify`?

Reply:
Sounds good!

Reply:
Addressed in https://github.com/netlify/edge-bundler/pull/427/commits/bf03f1992b301bb65d2e5862468824160854c109.